articles/141_static_remapping.md

```diff
@@ -324,8 +325,8 @@ The replacement must be a single token which will become the node's new name.
 
 Remapping rules are applied in the following order:
 
-1. Node name remapping
 1. Namespace remapping
+1. Node name remapping
```

Comment:
Changing the order doesn't seem like a good idea.

What needs to happen to resolve https://github.com/ros2/rcl/issues/296 is to allow the prefix to be a fully qualified name, instead of only a name.

In that way, if you have both `/asd/talker` and `/bsd/talker` in a node, you can do:

```
$ ros2 run package exec_name \
    --ros-args \
    -r /asd/talker:__node:=talker1 \
    -r /bsd/talker:__node:=talker2 \
    -r /asd/talker1:__ns:=/ns1 \   # or "-r talker1:__ns:=/ns1"
    -r /asd/talker2:__ns:=/ns2     # or "-r talker2:__ns:=/ns2"
```

+ in this way backwards compatibility isn't broken.

Reply:
Thank you for your response.

I know what you said, and also know that backward compatibility is important.
but there exists a limitation(actually, the example you provided is not what I want), it makes me feel uncomfortable.
Do you think we can remap the two node name in 'ros2 run examples_rclcpp_minimal_composition composition_composed' from ('publisher_node', 'subscriber_node') to ('/publisher/test_node', 'subscriber/test_node')?

Reply:
> What needs to happen to resolve ros2/rcl#296 is to allow the prefix to be a fully qualified name, instead of only a name.

this is true, and requirement is covered.

> but there exists a limitation(actually, the example you provided is not what I want)

sorry, could you describe the limitation stays after?

Reply:
The limitation is `remap the two node name in 'ros2 run examples_rclcpp_minimal_composition composition_composed' from ('publisher_node', 'subscriber_node') to ('/publisher/test_node', 'subscriber/test_node')`

Reply:
okay if the node names are the same, confliction comes in 1st place even with https://github.com/ros2/rcl/issues/296, thanks.

Reply:
> but there exists a limitation

I see, yeah my proposed solution doesn't solve the limitation pointed out here.
I think this proposal still has a limitation (see [here](https://github.com/ros2/design/pull/299#issuecomment-709330296)), and that is a problem I think users will more frequently run into.